test/pytest/test_sm_mme_requirements.py

```diff
@@ -0,0 +1,138 @@
+import os
+import subprocess
+import requests
+import pathlib
+import pytest
+import torch
+
+import test_utils
+
+CURR_FILE_PATH = os.path.dirname(os.path.realpath(__file__))
+REPO_ROOT = os.path.normpath(os.path.join(CURR_FILE_PATH, "..", ".."))
+MODELSTORE_DIR = os.path.join(REPO_ROOT, "model_store")
+data_file_kitten = os.path.join(REPO_ROOT, "examples/image_classifier/kitten.jpg")
+HF_TRANSFORMERS_EXAMPLE_DIR = os.path.join(REPO_ROOT, "examples/Huggingface_Transformers/")
+
+
+
+def download_transformer_model():
+    download_cmd = "cd " + HF_TRANSFORMERS_EXAMPLE_DIR + ";python Download_Transformer_models.py"
+    subprocess.check_call(download_cmd, shell=True)
+
+def create_transformer_mar_file():
+
+    extra_files = os.path.join(HF_TRANSFORMERS_EXAMPLE_DIR + "Transformer_model/config.json") + "," + \
+    os.path.join(HF_TRANSFORMERS_EXAMPLE_DIR, "setup_config.json") + "," + \
+    os.path.join(HF_TRANSFORMERS_EXAMPLE_DIR,"Seq_classification_artifacts/index_to_name.json")
+
+    pathlib.Path(test_utils.MODEL_STORE).mkdir(parents=True, exist_ok=True)
+
+    # Generate mar file
+    cmd = test_utils.model_archiver_command_builder(
+        model_name="BERTSeqClassification",
+        version="1.0",
+        serialized_file=os.path.join(HF_TRANSFORMERS_EXAMPLE_DIR, "Transformer_model/pytorch_model.bin"),
+        handler=os.path.join(HF_TRANSFORMERS_EXAMPLE_DIR, "Transformer_handler_generalized.py"),
+        extra_files=extra_files,
+        force=True,
+    )
+    cmd = cmd.split(" ")
+    subprocess.run(cmd, check=True)
+
+def test_no_model_loaded():
+    """
+    Validates that TorchServe returns reponse code 404 if no model is loaded.
+    """
+
+    os.makedirs(MODELSTORE_DIR, exist_ok=True)  # Create modelstore directory
+    test_utils.start_torchserve(model_store=MODELSTORE_DIR)
+
+    response = requests.post(url="http://localhost:8080/predictions/alexnet", data=open(data_file_kitten, 'rb'))
```

Comment:
please use API invoke since SM use it.
https://docs.aws.amazon.com/sagemaker/latest/dg/mms-container-apis.html

Reply:
@lxning Could you please point me to the invoke API example in TorchServe?